src/r1cs/linear_combination.rs

```diff
@@ -88,7 +88,7 @@ impl Sub<Variable> for Scalar {
 
     fn sub(self, other: Variable) -> Self::Output {
         LinearCombination {
-            terms: vec![(Variable::One(), self), (other, -Scalar::one())],
+            terms: vec![(Variable::One(), self), (other, -Scalar::ONE)],
         }
     }
 }
```

Comment:
Are we planning to pull in the linear proof changes for fixing the frozen heart vulnerability? https://github.com/dalek-cryptography/bulletproofs/pull/361

Reply:
I wasn't aware of this change. Should we be planning to pull in this change in this PR, or is it fine (i.e. less urgent) to do so in a follow-up PR?

Reply:
Talked with @jcape offline, and we'll pull in changes from upstream in a subsequent PR.